### DISCUSSION TITLE
unresolvedVariablesNotificationMode: warn does not work in sls v3

### DISCUSSION THREAD

**Discussion by cedric-fauth** [serverless/serverless#11199] [Q&A]
Hi, I have some variables which can't be resolved during development but that is not a problem because they are only used in production. I want serverless to ignore these errors and continue. In v2 it worked fine but with v3 `unresolvedVariablesNotificationMode: warn` does not work anymore and serverless raises errors:

```
Error:
Cannot resolve serverless.yml: Variables resolution errored with:
Cannot resolve variable at "custom.SECRET_KEY.prod": User: [...] is not authorized to perform: 
ssm:GetParameter on resource: [...] with an explicit deny in an identity-based policy
```

The error message is not important (in this case missing permissions), but serverless should ignore that like before.

**Comment by ptrhck** [comment]
Why is it resolving variables for a non-targeted stage anyway?
